tools/deployment/mmrotate_handler.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import base64
+import os
+
+import mmcv
+import torch
+from mmdet.apis import inference_detector, init_detector
+from ts.torch_handler.base_handler import BaseHandler
+
+import mmrotate  # noqa: F401
+
+
+class MMRotateHandler(BaseHandler):
+    threshold = 0.5
+
+    def initialize(self, context):
```

Comment:
Docstring is required, like-wise for `preprocess`, `inference` and `postprocess`.